aspects/thermal/network/bin/TdConvert/TdToGunnsXml.inc

```diff
@@ -83,14 +83,14 @@ C-----Warning issued if more conductors found than the size of the allocated arr
 HEADER OUTPUT CALLS, SPACE
 C-----Only print the post-run (steady state) data
       if (flag_print .eq. 0) then
-           flag_print = 1
-      else
-
+           flag_print = 1
+      else
+

```

Comment:
What actually changed here?  Just curious.  It's not showing any diff for me.

Reply:
I was able to see the difference by doing a `git diff` - there were `^M` characters at the end of those lines. Turns out those are carriage returns - at some point the file must have been edited in Windows, and my editor must be configured to remove them when I save files.
I'm not sure why this file was changed at all. Most likely, I ignored it in my whitespace clean-up because I didn't recognize the extension, but I have no idea why I would have saved the file this time.